modules/RTC/RTCUtils.js

```diff
@@ -869,17 +869,41 @@ class RTCUtils extends Listenable {
                 this.p2pPcConstraints
                     = JSON.parse(JSON.stringify(this.pcConstraints));
 
-                // Allows sending of video to be suspended if the bandwidth
-                // estimation is too low.
-                if (!options.disableSuspendVideo) {
-                    this.pcConstraints.optional.push(
-                        { googSuspendBelowMinBitrate: true });
+                const {
+                    forceSuspendVideo,
+                    disableSuspendVideo
+                } = options;
+                const suspendVideoEnabled
```

Comment:
i'm finding the meanings of `forceSuspendVideo` and `disableSuspendVideo` a bit confusing to figure out, i assume they'll be documented in config.js but is there any more explanation we could add here in a comment or something?

Reply:
I believe `disableSuspendVideo` is already documented. I was left with the impression that `forceSuspendVideo` will be something temporary for internal use only, that's why I prefer to not add it into the config. WDYT?